pipeline/relation_embedder/path_concatenator/src/main/scala/weco/pipeline/path_concatenator/PathsService.scala

```diff
@@ -0,0 +1,71 @@
+package weco.pipeline.path_concatenator
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.sksamuel.elastic4s.ElasticDsl._
+import com.sksamuel.elastic4s.circe._
+import com.sksamuel.elastic4s.streams.ReactiveElastic._
+import com.sksamuel.elastic4s.{ElasticClient, Index}
+import grizzled.slf4j.Logging
+import weco.json.JsonUtil._
+import akka.actor.ActorSystem
+import com.sksamuel.elastic4s.requests.searches.SearchRequest
+import weco.catalogue.internal_model.work.WorkState.Merged
+import weco.catalogue.internal_model.work.{CollectionPath, Work}
+import weco.json.JsonUtil.exportDecoder
+
+case class ParentPathData(collectionPath: CollectionPath);
+case class ParentPathState();
+case class PathHit(
+  data: ParentPathData
+)
+
+class PathsService(elasticClient: ElasticClient, index: Index)(
+  implicit as: ActorSystem)
+    extends Logging {
+  private val requestBuilder = new PathConcatenatorRequestBuilder(index)
+
+  /**
+    * Add a scroll to the given request and return an Elasticsearch Publisher
+    *
+    * The requests made by this service are expected to yield very few results
+    * getParentPath and getWorkWithPath should only ever return one result each.
+    *
+    * Setting up scroll context in Elasticsearch would appear to be an unnecessary
+    * overhead for this process, However, the Akka Source/publisher way of loading
+    * things requires a scroll query.
```

Comment:
Do we need to use the Akka Source/publisher approach? Is there any case where a vanilla query with a suitably large `size` parameter wouldn't work?

Reply:
(Good comment either way 👍)